From 870ecd8f982ebb97092a93d730ad4006bd78505c Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Wed, 8 Aug 2018 12:03:01 +0200
Subject: [PATCH] fix typo in flag value

---
 library/adenroll.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/adenroll.h b/library/adenroll.h
index f87dffa..abbbfd4 100644
--- a/library/adenroll.h
+++ b/library/adenroll.h
@@ -30,7 +30,7 @@ typedef enum {
 	ADCLI_ENROLL_NO_KEYTAB = 1 << 1,
 	ADCLI_ENROLL_ALLOW_OVERWRITE = 1 << 2,
 	ADCLI_ENROLL_PASSWORD_VALID = 1 << 3,
-	ADCLI_ENROLL_ADD_SAMBA_DATA = 1 << 3,
+	ADCLI_ENROLL_ADD_SAMBA_DATA = 1 << 4,
 } adcli_enroll_flags;
 
 typedef struct _adcli_enroll adcli_enroll;
-- 
2.17.1

